web/styles/app_variables.css

```diff
@@ -489,6 +489,9 @@
     --color-background-emoji-picker-emoji-reacted: hsl(195deg 50% 95%);
     --color-border-emoji-picker-emoji-reacted: hsl(195deg 52% 79%);
     --color-background-emoji-picker-emoji-reacted-focus: hsl(195deg 55% 88%);
+
+    /* Rendered markdown table color */
+    --border-rendered-markdown-tr-th-td: 1px solid hsl(0deg 0% 80%);
```

Comment:
This is looking better. Let's drop this down just to a color declaration, and follow the pattern of something like `--color-border-rendered-markdown-table`. You can then just reference that variable in the color syntax of the `border` property--rather than sandwiching all of those values into a single variable.